core/services/pipeline/task.bridge_test.go

```diff
@@ -353,6 +353,34 @@ func TestBridgeTask_DoesNotReturnStaleResults(t *testing.T) {
 
 	require.Error(t, result2.Error)
 	require.Nil(t, result2.Value)
+
+	task2 := pipeline.BridgeTask{
+		BaseTask:    pipeline.NewBaseTask(0, "bridge2", nil, nil, 0),
+		Name:        bridge.Name.String(),
+		RequestData: btcUSDPairing,
+		CacheTTL:    "35m", // more than the stalenessCap 30m
+	}
+	task2.HelperSetDependencies(cfg2, orm, specID, uuid.UUID{}, c)
+	// Insert entry 32s in the past, under cacheTTL of 35m but more than stalenessCap of 30m.
+	queryer.ExecQ(`INSERT INTO bridge_last_value(dot_id, spec_id, value, finished_at)
```

Comment:
Unchecked error?

Reply:
Added check for this and all other queries in the test